lib/api/CAnomalyJob.cc

```diff
@@ -972,6 +980,19 @@ bool CAnomalyJob::restoreDetectorState(const model::CSearchKey& key,
     return true;
 }
 
+bool CAnomalyJob::persistResidualModelsState(core::CDataAdder& persister,
+                                             ml::core_t::TTime timestamp) {
```

Comment:
We don't usually use the `ml::` qualifier in code that's in the `ml` namespace.

There are many other places where this could be removed.  It's generally only the stuff in `Main.cc` that's not in the `ml::` namespace.